src/test/java/uk/gov/pay/directdebit/mandate/services/MandateStateUpdateServiceTest.java

```diff
@@ -132,25 +132,13 @@ public void shouldRegisterEventWhenExchangingTokens() {
         assertThat(mandate.getState(), is(AWAITING_DIRECT_DEBIT_DETAILS));
     }
 
-    @Test
-    public void shouldUpdateMandateStateAndRegisterEventWhenConfirmingDirectDebitDetails_IfOneOff() {
-        Mandate oneOffMandate = MandateFixture
-                .aMandateFixture()
-                .withState(AWAITING_DIRECT_DEBIT_DETAILS)
-                .toEntity();
-        Mandate newMandate = service.confirmedOneOffDirectDebitDetailsFor(oneOffMandate);
-
-        assertThat(newMandate, is(oneOffMandate));
-        verify(mockedMandateDao).updateMandateReference(newMandate.getId(), oneOffMandate.getMandateReference());
-    }
-
     @Test
     public void shouldUpdateMandateStateAndRegisterEventWhenConfirmingDirectDebitDetails_andSendEmail_IfOnDemand() {
-        Mandate newMandate = service.confirmedOnDemandDirectDebitDetailsFor(onDemandMandate);
+        Mandate confirmedMandate = service.confirmedOnDemandDirectDebitDetailsFor(onDemandMandate);
 
-        assertThat(newMandate, is(onDemandMandate));
+        assertThat(confirmedMandate, is(onDemandMandate));
         verify(mockedUserNotificationService).sendOnDemandMandateCreatedEmailFor(onDemandMandate);
```

Comment:
can get rid of all references to "OnDemand" mandates here as there's no longer that distinction. Can be a separate PR